src/components/accordion/index.js

```diff
@@ -1,21 +1,36 @@
-import angular from 'core/angular';
-import './Accordion.scss';
+import angular from "core/angular";
+import "./Accordion.scss";
+class AccordionComponentController {
+    imageuri;
+    showimage;
+    ischecked;
+
+    onFailure() {
+        this.imageuri = undefined;
```

Comment:
What do you think to use null value instead undefined? 

I came with this question cuz "undefined" means that the variable doesn't have any reference. The compiler set undefined to all hoisted variables and when the code starts to run the variables will going to receive their proper values. 


```suggestion
        this.imageuri = null;
        this.showImage = null;
```

Reply:
Got it, it's just a pattern that is used on entire solution...